src/content/docs/style-guide/writing-docs/writer-workflow/tech-writer-workflow.mdx

```diff
@@ -157,4 +157,15 @@ To create a branch on the docs-website repo:
 3. Click on **New Branch**
 4. You will be prompted to name your new branch. Descriptive names are best. It's a great way to quickly clue people in to what your work is all about. For example, if you are working on What’s New pages, you might name the branch **Whats-new-updates**.
 
-When you create a new branch, don't forget to add the Jira ticket's key (DOC-1234) to the branch name and the PR title. 
\ No newline at end of file
+When you create a new branch, don't forget to add the Jira issue ID (DOC-1234) to the branch name and the PR title.
+
+## A note about GitHub permissions [#permissions]
```

Comment:
@paperclypse, I'm not really sure what problem this new section solves. If I'm a writer, how would I use this information about repository access?

Reply:
This is just meant to explain how GitHub permissions are currently set up for our repository. A couple of people expressed confusion about this so I thought I'd try to clarify that.

Reply:
@paperclypse since we're not telling writers which levers to pull in this area, I'm not sure it's going to be reassuring. Maybe we need to be more specific about when we might ever need to go in here and make changes?